src/ngraph/frontend/onnx_import/op/reverse_sequence.cpp

```diff
@@ -0,0 +1,48 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/op/reverse_sequence.hpp"
+#include "core/node.hpp"
+#include "ngraph/node.hpp"
+
+using namespace ngraph::op;
```

Comment:
It looks like you still use explicit full qualified name for classes, thus this is unnecessary.

Reply:
You are right. I removed unused declared napespace